decidim-core/app/views/decidim/account/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<main class="wrapper">
+<div class="row collapse">
+  <div class="columns">
+    <h1 class="heading1 user-header"><%= t(".title") %></h1>
+  </div>
+</div>
+<div class="row collapse">
+  <div class="main-container">
+    <div class="row collapse main-container--side-panel">
+      <div class="columns medium-4 large-3">
+        <div class="side-panel">
+          <ul class="tabs vertical side-panel__tabs" id="user-settings-tabs"
+            data-tabs>
+            <% if handlers.any? || authorizations.any? %>
+              <li class="tabs-title">
+                <a href="#authorizations"><%= t(".authorizations") %></a>
+              </li>
+            <% end %>
+          </ul>
+        </div>
+      </div>
+      <div class="columns medium-8 large-9">
+        <div class="main-container__content">
+          <div class="tabs-content vertical"
+            data-tabs-content="user-settings-tabs">
+            <div class="tabs-panel" id="authorizations">
+              <div class="row column">
+                <% if authorizations.any? %>
+                  <section class="section">
+                    <div class="card card--list">
+                      <% authorizations.each do |authorization| %>
+                        <div class="card--list__item">
+                          <div class="card--list__text">
+                            <%= icon "lock-unlocked", class: "card--list__icon" %>
+                            <div>
+                              <h5 class="card--list__heading">
+                                <%= t(authorization.name, scope: "decidim.authorization_handlers") %>
+                              </h5>
+                              <span class="text-small"><%= l(authorization.created_at, format: :long) %></span>
+                            </div>
+                          </div>
+                          <div class="card--list__data">
+                            <%= link_to authorization, method: :delete, class: "card--list__data__icon", data: { confirm: t(".authorization_confirm_destroy") } do %>
+                              <%= icon "circle-x" %>
+                            <% end %>
+                          </div>
+                        </div>
+                      <% end %>
+                    </div>
+                  <% end %>
+                  <% if handlers.any? %>
```

Comment:
Shouldn't you check if there's already an authorization for an existing handler, and exclude those? I don't imagine authorizing two times with the same handler (I don't think it's even possible with this codebase).